### DISCUSSION TITLE
Whip buff for NPC detected cheating false positive

### DISCUSSION THREAD

**Discussion by sgkoishi** [Pryaxis/TShock#1773] [General]
* TShock version: general-dev

#### Reproduction steps (if applicable)?

1. Use the whip (summoner weapon) to attack, inflict buff to NPCs
2. Get kicked for `Added buff to NPC abnormally.`

#### Any stack traces or error messages (if known)?
https://github.com/Pryaxis/TShock/blob/3471646dc71322ea4e71d224aa85c3e40eb9adf2/TShockAPI/Bouncer.cs#L1872-L1895
https://github.com/Pryaxis/TShock/blob/3471646dc71322ea4e71d224aa85c3e40eb9adf2/TShockAPI/Bouncer.cs#L1382-L1407

Whitelist not updated.

Well I'm not sure if this is 'big' enough to be an issue - just posting here as reminder

**Comment by Kojirremer** [comment]
https://github.com/Pryaxis/TShock/issues/1759
It's being worked on, so fingers crossed.

**Reply by sgkoishi** [reply]
Aha didn't see that! Lemme see if I can close or mark this duplicate.

**Comment by hakusaro** [comment]
This should be fixed now! Let me know if it's not!
